### DISCUSSION TITLE
Upgraded CopyQ to 9.0.1; help about still says 7.1.0

### DISCUSSION THREAD

**Discussion by Lockszmith-GH** [hluk/CopyQ#2875] [Q&A]
With UniGetUI, I've installed and updated CopyQ over winget.
Just installed 9.0.0, but Help About shows 7.1.0

What am I missing?

**Comment by hluk** [comment]
Did you restart the app and are you launching the newly installed app? BTW, the app can be installed from multiple sources (for example the distribution packages and Flatpak).

**Reply by Lockszmith-GH** [reply]
Looks like an issue with a single machine of mine. sorry for the noise. totally something on my end. I'll figure it out.
